src/Core/Domain/ValueObject/Email.php

```diff
@@ -86,7 +86,7 @@ public function isEqualTo(Email $email)
      */
     private function assertEmailIsValid($email)
     {
-        if (!filter_var($email, FILTER_VALIDATE_EMAIL)) {
+        if (!preg_match('/^.+\@.+$/', $email)) {
```

Comment:
Is this preg_match really useful?
Because "` azd    @  `"  will return true :thinking:

Reply:
@PierreRambaud Yeah, it will check that it looks like an email but it's not really trustworthy, maybe just an indication. Do you think we should get rid of this check is this ValueObject?

Reply:
I prefer having no check instead of a useless one :/